app/tests/test_collections_endpoint.py

```diff
@@ -89,9 +121,13 @@ def test_collections_limit_query(self):
 
     def test_valid_collections_post(self):
         payload_json = self.sample_collections['valid_collection_set_1']
+        grant_permissions('Collection', self.user)
+        self.client.login(username=self.username, password=self.password)
 
         response = self.client.post(
-            f"/{API_BASE}/collections", data=payload_json, content_type='application/json'
+            f"/{API_BASE}/collections",
+            data=json.dumps(payload_json),
+            content_type='application/json'
         )
         response_json = response.json()
         self.assertEqual(response.status_code, 201, msg=get_http_error_description(response_json))
```

Comment:
Mixin up PR is the door to the merging hell.

Reply:
That's nicely said :-) And true.
I could rebase on develop and remove all parts from #132. Then I'd have to add the unit tests on the POST/PUT endpoints later on in another PR.

Reply:
Actually you need to add the login to the POST/PUT in this PR as I've already merged POST/PUT/PATCH/DEL of Items ;-)